.readthedocs.yml

```diff
@@ -5,7 +5,7 @@ sphinx:
   configuration: doc/userdoc/conf.py
 
 conda:
-  environment: doc/environment.yml
+  environment: extras/conda-nest-dev.yml
```

Comment:
I wonder if we shouldn't rather go back to using the `requirements.txt` on Read the Docs, as that provides all that's needed, but not all that's not needed. The corresponding lines could be found [here](https://github.com/nest/nest-simulator/pull/1843/files#diff-3abe3227a1f802f5923ab34f41a3ad8e0d44d1395503a8d2da15a10ae42c1036L1-L6).

Reply:
I sent you https://github.com/heplesser/nest-simulator/pull/24. Hope this helps.

Reply:
Merged :)

Reply:
:+1: